src/cli/README_PERF.md

```diff
@@ -0,0 +1,480 @@
+# OpenThread CLI - Perf Example
+
+## Overview
+
+Cli Perf is a performance tool used to test throughput, latency, jitter, out of order and loss rate of OpenThread Network.
+
+## Quick Start
+
+### Build with Time Sync and Cli Perf support
+
+Use the `TIME_SYNC=1` build switch to enable Time Sync API support.
+Use the `CLI_PERF=1` build switch to enable Cli Perf support.
+
+Note: Latency measurement is based on Time Sync feature.
+
+```bash
+> ./bootstrap
+> make -f examples/Makefile-nrf52840 TIME_SYNC=1 CLI_PERF=1 
+```
+
+### Example 1: Test Unicast Traffic
+
+#### Form Network and enable Time Sync
+
+Form a network with at least two devices. All devices are time synchronized with each other.
+
+#### Node 1
+
+On node 1, create a listening session and start it.
+
+```bash
+> perf server
```

Comment:
is it better to mimic `iperf` syntax? @jwhui

Reply:
The perf commands follow the OpenThread cli command style and I think it is better for us to remember and use.